app/src/main/java/com/ataulm/artcollector/ApplicationComponent.kt

```diff
@@ -0,0 +1,22 @@
+package com.ataulm.artcollector
+
+import com.squareup.picasso.Picasso
+import dagger.BindsInstance
+import dagger.Component
+
+@Component(modules = [ApplicationModule::class])
+interface ApplicationComponent {
+
+    fun picasso(): Picasso
+
+    fun harvardApiMuseumApi(): HarvardArtMuseumApi
```

Comment:
we explicitly declare the objects that this component provides, so that any other component that depends on this one can see them.

without this, dagger doesn't know where to find `HarvardArtMuseumApi` which is needed in the `PaintingsModule`.